studyBuddy/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -142,13 +146,15 @@ private Entity createStudentEntity(Student student){
 *****************************************************/
   private Student createStudentFromEntity(Entity theStudentEntity){
       final String nickname = (String)theStudentEntity.getProperty("nickname");
```

Comment:
Since the first and last names have been added, I think we can not get rid of this nickname field! right?

Reply:
The reason I didn't remove it is to maintain backward compatibility. Any requests made before this gets pushed will not have the first and last name attached to it. If we deprecate the field, these cases will cause errors when being fetched. If we do want to remove these fields, we can either back fill the previous entries, or since they are all just test entries, we could just remove them all.